CONTRIBUTING.md

```diff
@@ -0,0 +1,24 @@
+# Contribution guidelines
+
+Thank you for your interest in contributing. It is greatly appreciated.
+
+# Pull requests
+
+Code contributions to this project follow this process and set of requirements:
+
+1. As a contributor you should [create an issue](http://platform.dandigbib.org/projects/ddb-cms/issues/new) describing the suggested change on [our Redmine installation platform.dandigbib.org](http://platform.dandigbib.org/projects/ddb-cms/) in Danish. This allows the business side of the project to evaluate your suggestion. If it accepted for inclusion it will be marked for review. To create an issue you must [have an account on the site](http://platform.dandigbib.org/account/register). The only exception from this rule is if the change does not affect end users in any way. This could be developer documentation changes, technical tests or changes to tools.
+2. All communication on GitHub should be in English.
+3. The title of your pull request must refer to the Redmine issue number if applicable e.g. *#[issue number]: [short summary of change]*.
+4. The description of your pull request should include a link to the issue.
+5. The body of the pull request should provide a short description of the suggested change and the reasoning behind the approach you have chosen.
```

Comment:
It's standard behaviour: `git commit` will start an editor on an file with # comments. That's why it ended up as "Issue #" in the Drupal guidelines when they moved from CVS.

That being said, most commit message guidelines I've seen does not  suggest putting the issue reference at the beginning of the first line, but rather later. In most cases when a developer is looking at commit logs, the issue number is irrelevant (who can remember what bug 3187 was?), while the description is more informative. And the issue number eats up of the character limit.

Secondly, there's a difference between pull requests descriptions and commit messages.